suite-common/wallet-core/src/fiat-rates/fiatRatesThunks.ts

```diff
@@ -53,7 +53,7 @@ export const updateTxsFiatRatesThunk = createThunk(
         const groupedTokensTxs = groupTokensTransactionsByContractAddress(txs);
 
         for (const token in groupedTokensTxs) {
-            const hasCoinDefinitions = getNetworkFeatures(account.symbol as NetworkSymbol).includes(
+            const hasCoinDefinitions = getNetworkFeatures(account.symbol).includes(
```

Comment:
incidental easy fix, not related to `isArrayMember` at all :slightly_smiling_face: